packages/grafana-ui/src/components/Table/styles.ts

```diff
@@ -82,6 +82,7 @@ export const getTableStyles = stylesFactory(
 
         &:last-child {
           border-right: none;
+          padding-right: 5px;
```

Comment:
nit: shouldn't use spacing from the theme instead of hardcoding values? Or maybe that is hard to use in this context?

Reply:
Yeah, we can. Just tried with 8px and it looks ok.